### PR TITLE
Update observability tests

### LINKED ISSUE #289: Remove Observability flaky tests from FTR
### Is your feature request related to a problem?
Some of the tests/test suits for Observability plugin are flaky which causes false alarms for releases from time to time.

### What solution would you like?
Remove unnecessary but prone-to-be-flaky tests, and fix/improve those that are necessary.

### PR DESCRIPTION
**Pull request by mengweieric**
### Description

Removed/updated observability tests.

### Issues Resolved

#289 

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/plugins/observability-dashboards/7_app_analytics.spec.js [hunk 1]
```diff
@@ -453,6 +254,7 @@ describe('Application Analytics home page', () => {
       .within(() => {
         cy.get('.euiCheckbox').click();
       });
+    cy.wait(delayTime);
```
Comment (ashwin-pc) [inline cypress/integration/plugins/observability-dashboards/7_app_analytics.spec.js:257]: Isn't there a better way to implement this? `wait(ms)` should be almost universally avoided. The sony exception i can think of is when debouncing is involved.

**Comment by ashwin-pc** [comment]
@mengweieric looks like your tests dont pass the CI. Also can you attach the video for the updated tests?

**Comment by mengweieric** [comment]
> @mengweieric looks like your tests dont pass the CI. Also can you attach the video for the updated tests?

Sure later will update PR to address the issue and upload video.

**Comment by mengweieric** [comment]
Test videos

https://user-images.githubusercontent.com/80358241/211919519-8c67b6e9-4a7d-4ad1-bad8-b6c4923c046d.mp4


https://user-images.githubusercontent.com/80358241/211919535-88529d17-3c02-43e8-ba2a-f10ceca8ea8d.mp4


https://user-images.githubusercontent.com/80358241/211919542-6768d8c4-21e4-4d06-bdf6-2c3a9e3eb6b3.mp4


https://user-images.githubusercontent.com/80358241/211919547-209b4a52-8ebc-4955-b3d7-f40baf51f174.mp4


https://user-images.githubusercontent.com/80358241/211919551-40752755-8f6c-43ed-9eb1-1939265ba854.mp4


https://user-images.githubusercontent.com/80358241/211919568-797068ff-47fb-445f-96c5-4426169d7c79.mp4


https://user-images.githubusercontent.com/80358241/211919608-9435344e-1f3a-4080-8014-93e86a174354.mp4

**Comment by mengweieric** [comment]
Screen Shot 2023-01-11 at 1 20 09 PM

**Review by tianleh — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**